cms/static/sass/programs/_app-typography.scss

```diff
@@ -0,0 +1,80 @@
+
+// ------------------------------
+// Programs: App Fonts
+
+// About: resolving issue with rem units when html font-size is not 100%.
+// the issue is that the unit being used for fonts, padding and margins in the
+// UX Pattern Library (UXPL) is relative to the font-size set on the html element.
+// In studio this is set to 62.5%, however the UXPL assumes it is set to 100%. Because
+// the page in studio has other elements that are native to studio that need to be styled
```

Comment:
Haven't you taken care of this as part of this PR? Are these typography overrides still required?